crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -67,18 +67,37 @@ def services
   api :POST, "/api/upgrade/controllers", "Orchestrate the upgrade of controller nodes"
   api_version "2.0"
   def controllers
+    status = :ok
+    msg = ""
+
     # First node to upgade is DRBD slave
-    # FIXME: this actually does all drbd slaves in all clusters, do we want that?
-    NodeObject.find("drbd_rsc_postgresql_master:true").each do |node|
-      node_api = Api::Node.new node.name
-      next if node_api.upgraded?
-      node_api.pre_upgrade
-      node_api.upgrade
-      node_api.post_upgrade
-      # FIXME: check return values if it makes sense
+    # FIXME: what about other drbd slaves?
+    node = NodeObject.find("drbd_rsc_postgresql_master:true").first
```

Comment:
Is this change really related to "Check for return values of ssh commands"? I don't think it belongs in this commit. Should be squashed into the commit it depends on, since it seems to be part of transient development.